docs/content/examples/tutorials/basic-installation.md

```diff
@@ -84,142 +79,3 @@ services:
-              - "465:465"
-            volumes:
-              - ./docker-data/dms/mail-data/:/var/mail/
-              - ./docker-data/dms/mail-state/:/var/mail-state/
-              - ./docker-data/dms/mail-logs/:/var/log/mail/
-              - ./docker-data/dms/config/:/tmp/docker-mailserver/
-              - /etc/localtime:/etc/localtime:ro
-            environment:
-              - ENABLE_FAIL2BAN=1
-              # Using letsencrypt for SSL/TLS certificates
-              - SSL_TYPE=letsencrypt
-              # Allow sending emails from other docker containers
-              # Beware creating an Open Relay: https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#permit_docker
-              - PERMIT_DOCKER=network
-              # You may want to enable this: https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#spoof_protection
-              # See step 8 below, which demonstrates setup with enabled/disabled SPOOF_PROTECTION:
-              - SPOOF_PROTECTION=0
-            cap_add:
-              - NET_ADMIN # For Fail2Ban to work
-            restart: always
-        ```
-
-    - The docs have a detailed page on [Environment Variables][docs-environment] for reference.
-
-    !!! note "Firewalled ports"
-
-        You may need to open ports `25`, `587` and `465` on the firewall. For example, with the firewall `ufw`, run:
-
-        ```sh
-        ufw allow 25
-        ufw allow 587
-        ufw allow 465
-        ```
-
-4. Configure your DNS service to use an MX record for the _hostname_ (eg: `mail`) you configured in the previous step and add the [SPF][docs-spf] TXT record.
-
-    If you manually manage the DNS zone file for the domain, it would look something like this:
-
-    ```txt
-    mail      IN  A   10.11.12.13
-
-    ; mail-server for example.com
-        3600  IN  MX  1  mail.example.com.
-
-    ; Add SPF record
-              IN TXT "v=spf1 mx ~all"
-    ```
-
-    Then don't forget to change the serial number and to restart the service.
-
-5. [Generate DKIM keys][docs-dkim] for your domain via `./setup.sh config dkim`.
-
-    Copy the content of the file `docker-data/dms/config/opendkim/keys/example.com/mail.txt` and add it to your DNS records as a TXT like SPF was handled above.
-
-    I use [bind9](https://github.com/docker-scripts/bind9) for managing my domains, so I just paste it on `example.com.db`:
-
-    ```txt
-    mail._domainkey IN      TXT     ( "v=DKIM1; h=sha256; k=rsa; "
-            "p=MIIBIjANBgkqhkiG9w0BAQEFACAQ8AMIIBCgKCAQEAaH5KuPYPSF3Ppkt466BDMAFGOA4mgqn4oPjZ5BbFlYA9l5jU3bgzRj3l6/Q1n5a9lQs5fNZ7A/HtY0aMvs3nGE4oi+LTejt1jblMhV/OfJyRCunQBIGp0s8G9kIUBzyKJpDayk2+KJSJt/lxL9Iiy0DE5hIv62ZPP6AaTdHBAsJosLFeAzuLFHQ6USyQRojefqFQtgYqWQ2JiZQ3"
-            "iqq3bD/BVlwKRp5gH6TEYEmx8EBJUuDxrJhkWRUk2VDl1fqhVBy8A9O7Ah+85nMrlOHIFsTaYo9o6+cDJ6t1i6G1gu+bZD0d3/3bqGLPBQV9LyEL1Rona5V7TJBGg099NQkTz1IwIDAQAB" )  ; ----- DKIM key mail for example.com
-    ```
```

Comment:
Similarly, this one is focused on a "happy path".

The `usage.md` equivalent section is good, but it's explaining three different scenarios (_rspamd vs opendkim support, and LDAP advice_). I know we've already discussed that the DKIM docs can be improved in a future PR though :+1:

Reply:
Forgot about my suggestion for DKIM in `usage.md`, applied that and it's improved this concern for the most part.